Code/PgSQL/rdkit/adapter.cpp

```diff
@@ -2108,7 +2111,7 @@ extern "C" char *findMCSsmiles(char *smiles, char *params) {
   while (*s && *s <= ' ') {
     s++;
   }
-  while (s < s_end && *s > ' ') {
+  while (s<s_end && * s> ' ') {
```

Comment:
```suggestion
  while (s < s_end && *s > ' ') {
```
This change seems unintentional (`clang-format` gone wrong?)

Reply:
Indeed. Thanks for noticing that.